pyproject.toml

```diff
@@ -1,16 +1,16 @@
 [tool.black]
 line-length = 160
-target-version = ['py37']
+target-version = ['py39']
 
 [tool.isort]
 profile = "black"
 line_length = 160
 
 [tool.mypy]
-python_version = 3.7
+python_version = 3.9
```

Comment:
Can we not make changes in between releases to any packages please? Trying to keep the CI system from exploding.

Reply:
Reverted now, thanks.